target/linux/ramips/dts/mt7620a_engenius_esr600.dts

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "engenius,esr600", "ralink,mt7620a-soc";
+	model = "EnGenius ESR600";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
```

Comment:
I think memory node can be dropped.

Reply:
I'm away from the device until after 17 August, but I will make that change and  test as soon as possible after that.

Reply:
I can confirm that it auto-detects the correct memory size without the "memory@0" node in the DTS.  I will remove that.